### PR TITLE
add debug() in api and larray namespace

### PR DESCRIPTION
**Pull request by gdementen**

### CODE REVIEW

File: larray/inout/pickle.py [hunk 1]
```diff
@@ -55,5 +55,6 @@ def _dump_metadata(self, metadata):
         self.data['__metadata__'] = metadata
 
     def close(self):
+        # FIXME: this is wrong in so many ways. If we only read the file, it will be re-dumped when closed?
```
Comment (alixdamman) [inline larray/inout/pickle.py]: why not override the save() method and move the code below to it? Instead of adding a FIXME.

**Comment by alixdamman** [comment]
Except for the FIXME (open a new issue for it?), LGTM.

File: larray/inout/pickle.py (hunk 1 above)
Comment (gdementen) [reply larray/inout/pickle.py]: because I didn't write that code and don't know the implications of modifying it. If it's that simple, I will try that.

**Comment by gdementen** [comment]
It wasn't as simple as that to fix it properly, but not too hard either :). Merging...

**Pull request merged**